### DISCUSSION TITLE
Report

### DISCUSSION THREAD

**Discussion by Kashmir00** [dcajasn/Riskfolio-Lib#195] [Q&A]
Hi, I get this error trying to plot Jupyter report: ValueError: 
Normal: $\mu=0.18%$%, $\sigma=1.85%$%
        ^
ParseException: Expected end of text, found '$'  (at char 8), (line:1, col:9)

<Figure size 1008x2592 with 6 Axes>

**Comment by dcajasn** [comment]
Update Riskfolio-Lib and Matplotlib
